packages/flutter_tools/lib/src/web/web_device.dart

```diff
@@ -218,7 +218,7 @@ class WebServerDevice extends Device {
 
   @override
   DeviceLogReader getLogReader({ApplicationPackage app}) {
-    return _logReader ??= NoOpDeviceLogReader(app.name);
+    return _logReader ??= NoOpDeviceLogReader(app?.name);
```

Comment:
Is it okay to leave null. Maybe `app?.name ?? 'name'`?

Reply:
We don't actually do anything with the name for the no-op version: https://github.com/flutter/flutter/blob/bda9d90804f7b592d5405a621836f4cec1ebe2a2/packages/flutter_tools/lib/src/device.dart#L674

🤞 NNBD save us 🤞